static/app/components/comboBox/index.tsx

```diff
@@ -464,4 +479,11 @@ const StyledLoadingIndicator = styled(LoadingIndicator)`
     width: 12px;
   }
 `;
+
+const MenuFooter = styled('div')`
```

Comment:
I copied this from [this file on GitHub](https://github.com/getsentry/sentry/blob/3ee19817686bed81ec94843d05bef798db7e21f6/static/app/components/compactSelect/control.tsx#L708). I chose not to export the style because it felt not correct to do so, but I would still like to find a way to reuse it. @vuluongj20 do you have an opinion?